frontend/src/pages/Profile/UserSkills/SkillInfo.tsx

```diff
@@ -27,23 +28,35 @@ const CourseMaterialInfo: React.FC<Props> = ({
   stagesCompleted,
   progress,
   imageUrl,
-}) => (
-  <SkillInfo completed={stages === stagesCompleted}>
-    <SkillInfoFlex>
-      <ImageWrapper>
-        <UserAvatar size={SIZE.small} avatar={imageUrl} />
-      </ImageWrapper>
-      <div style={{ width: '70px' }}>
-        <SkillInfoText>{technologyTitle}</SkillInfoText>
-        <SkillInfoStage>
-          {stagesCompleted}/{stages}
-        </SkillInfoStage>
-      </div>
-      <SkillProgress>
-        <ProgressBar value={progress} size={SIZE.small} />
-      </SkillProgress>
-    </SkillInfoFlex>
-  </SkillInfo>
-);
+}) => {
+  const [isShown, setIsShown] = useState(false);
+
+  return (
+    <SkillInfo completed={stages === stagesCompleted}>
+      <SkillInfoFlex>
+        <ImageWrapper>
+          <UserAvatar size={SIZE.small} avatar={imageUrl} />
+        </ImageWrapper>
+        <div style={{ position: 'relative' }}>
+          <div style={{ width: '70px' }}>
+            <SkillInfoText
+              onMouseEnter={() => setIsShown(true)}
```

Comment:
Maybe you should create separate functions (in SkillsInfoContainer) and reuse them here like this:
`onMouseEnter={showSkillInfo}`